include/segments.h

```diff
@@ -11,6 +11,7 @@
  * linker script syntax.
 */
 
+#define USE_EXT_RAM
```

Comment:
i did not plan to have this be a repo that would be used in the future - but if this is supposed to be a repo about removing optimizations to make stuff run faster, we can't just add EXT_RAM support since that would make the game run faster and be a new optimization.